TELCODOCS-1526: Added a documentation bug fix section to complete change management. (#65116)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

* Previously, the *Samples* page in the {product-title} did not allow distinguishing between the types of samples listed. With this fix, you can identify the sample from the badges displayed on the *Samples* page. (link:https://issues.redhat.com/browse/OCPBUGS-10679[*OCPBUGS-10679*])

[discrete]
[id="ocp-4-13-documentation-bug-fixes"]
==== Documentation

> Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation.

Reviewer: ```suggestion
Previously, the {product-title} documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up to date documentation.
```

> Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation."

Reviewer: ```suggestion
Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation.
```

Reviewer: Removed quotes from original suggestion. Please update and I will remove/add necessary labels.

> * Previously, the {product-title} documentation contained a "Expanding a cluster with on-premise bare metal nodes" sub section under the "Installing on AWS" section, because it was not accurate. The removal of this documentation is intended to avoid confusion.

Reviewer: ```suggestion
Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation."
